front_end/panels/ai_chat/evaluation/framework/GenericToolEvaluator.ts

```
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 import type { Tool } from '../../tools/Tools.js';
```

ESLint

[error] '../../tools/Tools.js' imported multiple times.

(import/no-duplicates)